uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/dom/single/impl/BaseSingletonDOMElementFactory.java

```diff
@@ -81,19 +80,21 @@ public void execute() {
     public E createDomElement(final GridLayer gridLayer,
                               final GridWidget gridWidget,
                               final GridBodyCellRenderContext context) {
-        this.widget = createWidget();
-        this.widget.addDomHandler(new KeyDownHandlerCommon(gridPanel, gridLayer, gridWidget, this, context),
-                                  KeyDownEvent.getType());
-        this.e = createDomElementInternal();
-        widget.addDomHandler((e) -> e.stopPropagation(), MouseDownEvent.getType());
-        widget.addDomHandler(new BlurHandler() {
-            @Override
-            public void onBlur(final BlurEvent event) {
-                flush();
-                gridLayer.batch();
-                gridPanel.setFocus(true);
-            }
-        }, BlurEvent.getType());
+        widget = createWidget();
+        e = createDomElementInternal();
+
+        widget.addDomHandler(new KeyDownHandlerCommon(gridPanel, gridLayer, gridWidget, this, context),
```

Comment:
Could you move the instantiation of `KeyDownHandlerCommon` to a `protected` method; so when I change DMN for [this](https://github.com/kiegroup/appformer/pull/684) I only need to override the new method to return an instance that does not handle `KeyCodes.KEY_ENTER`?

Reply:
Done, please have a look, if it is what you need.